example_envs/bookinfo/main.rs

```diff
@@ -110,7 +110,7 @@ fn main() {
 
     // Execute the simulator
     simulator.insert_rpc("gateway", Rpc::new("0"));
-    for tick in 0..6 {
+    for tick in 0..7 {
```

Comment:
You should also add a test for this in the integration test suite we have. Main is not really used at all.

Reply:
Yeah, I'm getting this fixed, then I'll get the compiler fixed, and then we'll have big integration tests.

Reply:
Oh I meant the tests in the tests folder. This is likely a 3 line change. Main should not be used for testing at all. It is just an example.

Reply:
Oh I see.  Yeah, that's a good idea.  Should be in there now.